sources/packages/backend/workflow/test/models/assessment.model.ts

```diff
@@ -234,13 +236,18 @@ export interface CalculatedAssessmentModel {
   finalProvincialAwardNetSBSDAmount: number;
   // Calculated Data
   calculatedDataTotalRemainingNeed3: number;
+  // Calculated Data
```

Comment:
Is it not the same comment from line 237? Can we have them grouped and ordered as below?
```ts
// Calculated Data
calculatedDataTotalRemainingNeed1: number;
calculatedDataTotalRemainingNeed3: number;

```

Reply:
I noticed that was changed as below
```ts
calculatedDataTotalRemainingNeed3: number;
calculatedDataTotalRemainingNeed1: number;
```
when the request was to have it as 
```ts
calculatedDataTotalRemainingNeed1: number;
calculatedDataTotalRemainingNeed3: number;
```
I am resolving the comment because it is minor.